muxink/src/codec/bincode.rs

```diff
@@ -93,7 +99,10 @@ where
         let (outcome, consumed) = {
             let slice: &[u8] = buffer.as_ref();
             let mut cursor = Cursor::new(slice);
-            let outcome = bincode::deserialize_from(&mut cursor);
+            let outcome = DefaultOptions::new()
```

Comment:
We could standardize on one set of `bincode` encoding options here even. Jury's still out on whether fixint or varint, I hope the latter will work.

Reply:
Currently, the varint is used.